README.md

```diff
@@ -40,7 +38,7 @@ Wow, looks good right? Here you can configure all your mods, even ones that don'
 
 Welcome!
 
-Here you can find how to create Minecraft mods with OneConfig, create configurations for your mods that automatically save and can be edited from the OneConfig GUI, create commands and your own GUI using [oneuiscreen.md](gui/oneuiscreen.md "mention") and much, much more!
+Here you can find how to create Minecraft mods with OneConfig, create configurations for your mods that automatically saved and can be edited from the OneConfig GUI, create commands and your own GUI using [oneuiscreen.md](gui/oneuiscreen.md "mention") and much, much more!
```

Comment:
no?

Reply:
that automatically saved is not correct

Reply:
the other way doesn't sound correct either even if it is
maybe `create auto-saving configurations for your mods`

Reply:
or `Here you can find how to create Minecraft mods with OneConfig, with configuration that's auto-saving and can be edited from the OneConfig menu, commands to do things, a custom UI, and much more!`

Reply:
^ kti version is better